tests/grate/asm/fs_vs_tests.txt

```diff
@@ -0,0 +1,13 @@
+alu_buffer_size = 1
+pseq_to_dw_exec_nb = 1
+
+.asm
+EXEC
+	MFU:	var unk(0x104E51BA0) r0.fp20, r0.fp20, r0.fp20, r0.fp20
```

Comment:
@digetx: Ah, I didn't yet see that. I was hoping for something like that, but my earlier experiments didn't reveal such a bit. I guess I made a mistake, this is much nicer!